### PR TITLE
Make time-related variables more readable

### PR DESCRIPTION
**Pull request by asdf2014**
Make time-related variables more readable

### CODE REVIEW

**Comment by asdf2014** [comment]
It seems that the failure of those jobs is not related to this PR. Can someone please help me to restart them? (job#[415047952](https://travis-ci.org/apache/incubator-druid/jobs/415047952) / job#[415047956](https://travis-ci.org/apache/incubator-druid/jobs/415047956))

File: common/src/main/java/io/druid/common/config/Log4jShutdown.java [hunk 1]
```diff
@@ -28,11 +28,12 @@
 import javax.annotation.concurrent.GuardedBy;
 import java.util.Queue;
 import java.util.concurrent.ConcurrentLinkedQueue;
+import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicBoolean;
 
 public class Log4jShutdown implements ShutdownCallbackRegistry, LifeCycle
 {
-  private static final long SHUTDOWN_WAIT_TIMEOUT = 60000;
+  private static final long SHUTDOWN_WAIT_TIMEOUT = TimeUnit.MINUTES.toMillis(1);
```
Comment (clintropolis) [inline common/src/main/java/io/druid/common/config/Log4jShutdown.java]: This is nice, but any chance you could be convinced to also rename these variables to include the unit of time they represent? e.g. `SHUTDOWN_WAIT_TIMEOUT_MILLIS` instead fo `SHUTDOWN_WAIT_TIMEOUT`

File: indexing-service/src/main/java/io/druid/indexing/overlord/TaskQueue.java [hunk 2]
```diff
@@ -290,7 +291,7 @@ public String apply(Task task)
         }
         // awaitNanos because management may become necessary without this condition signalling,
         // due to e.g. tasks becoming ready when other folks mess with the TaskLockbox.
-        managementMayBeNecessary.awaitNanos(60000000000L /* 60 seconds */);
+        managementMayBeNecessary.awaitNanos(TimeUnit.SECONDS.toNanos(60));
```
Comment (clintropolis) [inline indexing-service/src/main/java/io/druid/indexing/overlord/TaskQueue.java]: Might be nice to pull this out into a constant if you're feeling motivated

File: common/src/main/java/io/druid/common/config/Log4jShutdown.java (hunk 1 above)
Comment (asdf2014) [reply common/src/main/java/io/druid/common/config/Log4jShutdown.java]: Hi, @clintropolis . Thanks for your comments. It's has been done.

File: indexing-service/src/main/java/io/druid/indexing/overlord/TaskQueue.java (hunk 2 above)
Comment (asdf2014) [reply indexing-service/src/main/java/io/druid/indexing/overlord/TaskQueue.java]: Done.

File: common/src/main/java/io/druid/common/config/Log4jShutdown.java (hunk 1 above)
Comment (clintropolis) [reply common/src/main/java/io/druid/common/config/Log4jShutdown.java]: Oops, I didn't mean just this variable, I meant all of the ones that were changed. Sorry for the confusion!

Comment (asdf2014) [reply common/src/main/java/io/druid/common/config/Log4jShutdown.java]: I just also realized it, yes, I will fix them all.

Comment (asdf2014) [reply common/src/main/java/io/druid/common/config/Log4jShutdown.java]: Done. :sweat_smile:

**Comment by jihoonson** [comment]
@asdf2014 thanks! I've left some comments and restarted the failed jobs.

**Comment by clintropolis** [comment]
Nice! :+1: after changes @jihoonson suggested

File: extensions-contrib/ambari-metrics-emitter/src/main/java/io/druid/emitter/ambari/metrics/AmbariMetricsEmitterConfig.java [hunk 3]
```diff
@@ -25,13 +25,13 @@
...
 import java.util.List;
-
+import java.util.concurrent.TimeUnit;
 
 public class AmbariMetricsEmitterConfig
 {
   private static final int DEFAULT_BATCH_SIZE = 100;
-  private static final Long DEFAULT_FLUSH_PERIOD_MILLIS = (long) (60 * 1000); // flush every one minute
-  private static final long DEFAULT_GET_TIMEOUT = 1000; // default wait for get operations on the queue 1 sec
+  private static final Long DEFAULT_FLUSH_PERIOD_MILLIS = TimeUnit.MINUTES.toMillis(1); // flush every one minute
```
[the remark this answers is not in the document]
Comment (asdf2014) [reply extensions-contrib/ambari-metrics-emitter/src/main/java/io/druid/emitter/ambari/metrics/AmbariMetricsEmitterConfig.java]: Done.

File: extensions-contrib/graphite-emitter/src/main/java/io/druid/emitter/graphite/GraphiteEmitterConfig.java [hunk 4]
```diff
@@ -25,15 +25,15 @@
...
+import java.util.concurrent.TimeUnit;
 
 public class GraphiteEmitterConfig
 {
   public static final String PLAINTEXT_PROTOCOL = "plaintext";
   public static final String PICKLE_PROTOCOL = "pickle";
   private static final int DEFAULT_BATCH_SIZE = 100;
-  private static final Long DEFAULT_FLUSH_PERIOD = (long) (60 * 1000); // flush every one minute
-  private static final long DEFAULT_GET_TIMEOUT = 1000; // default wait for get operations on the queue 1 sec
+  private static final Long DEFAULT_FLUSH_PERIOD_MILLIS = TimeUnit.MINUTES.toMillis(1); // flush every one minute
```
[the remark this answers is not in the document]
Comment (asdf2014) [reply extensions-contrib/graphite-emitter/src/main/java/io/druid/emitter/graphite/GraphiteEmitterConfig.java]: Done.

File: extensions-contrib/kafka-eight-simpleConsumer/src/main/java/io/druid/firehose/kafka/KafkaEightSimpleConsumerFirehoseFactory.java [hunk 5]
```diff
@@ -72,7 +73,7 @@
 
   private final List<PartitionConsumerWorker> consumerWorkers = new CopyOnWriteArrayList<>();
   private static final int DEFAULT_QUEUE_BUFFER_LENGTH = 20000;
-  private static final int CONSUMER_FETCH_TIMEOUT = 10000;
+  private static final int CONSUMER_FETCH_TIMEOUT_MILLIS = (int) TimeUnit.SECONDS.toMillis(10);
```
[the remark this answers is not in the document]
Comment (asdf2014) [reply extensions-contrib/kafka-eight-simpleConsumer/src/main/java/io/druid/firehose/kafka/KafkaEightSimpleConsumerFirehoseFactory.java]: Yes, I have tried. But, `io.druid.firehose.kafka.KafkaSimpleConsumer#fetch` method needs an `int` type `timeoutMs` param.

File: extensions-contrib/kafka-eight-simpleConsumer/src/main/java/io/druid/firehose/kafka/KafkaSimpleConsumer.java [hunk 6]
```diff
@@ -74,10 +75,10 @@
   private List<HostAndPort> replicaBrokers;
   private SimpleConsumer consumer = null;
 
-  private static final int SO_TIMEOUT = 30000;
+  private static final int SO_TIMEOUT_MILLIS = (int) TimeUnit.SECONDS.toMillis(30);
```
[the remark this answers is not in the document]
Comment (asdf2014) [reply extensions-contrib/kafka-eight-simpleConsumer/src/main/java/io/druid/firehose/kafka/KafkaSimpleConsumer.java]: Yes, I have tried it. But, the constructor of `SimpleConsumer` needs an `int` type `soTimeout` parameter.

File: processing/src/test/java/io/druid/query/AsyncQueryRunnerTest.java [hunk 7]
```diff
@@ -38,7 +38,7 @@
 public class AsyncQueryRunnerTest
 {
 
-  private static final long TEST_TIMEOUT = 60000;
+  private static final long TEST_TIMEOUT_MILLIS = 60_000;
```
[the remark this answers is not in the document]
Comment (asdf2014) [reply processing/src/test/java/io/druid/query/AsyncQueryRunnerTest.java]: Yep, I have tried this too. However, the `@Test` annotation requires a constant attribute value.

**Pull request merged**